fsButler/utils.py

```diff
@@ -9,14 +10,15 @@
 import lsst.analysis.utils as utils
 import lsst.afw.display.ds9 as ds9
 
-from lsst.pex.exceptions import LsstCppException
-from lsst.meas.extensions.multiShapelet import FitPsfModel
+# from lsst.pex.exceptions import LsstCppException
+from lsst.pex.exceptions import Exception as LsstCppException
+# from lsst.meas.extensions.multiShapelet import FitPsfModel
 
 """
 Utility functions to process data elements delivered by fsButler
 """
 
-#TODO: Make these configurable options
+# TODO: Make these configurable options
```

Comment:
(Here and throughout) whitespace churn should be on a separate commit, if, indeed, it's really necessary at all.